plugin/trino-hive/src/main/java/io/trino/plugin/hive/coercions/DecimalCoercers.java

```diff
@@ -228,6 +233,55 @@ protected void applyCoercedValue(BlockBuilder blockBuilder, Block block, int pos
         }
     }
 
+    public static Function<Block, Block> createDecimalToVarcharCoercer(DecimalType fromType, VarcharType toType)
+    {
+        int lengthLimit = toType.getLength().orElse(Integer.MAX_VALUE);
```

Comment:
Use `io.trino.spi.type.VarcharType#UNBOUNDED_LENGTH` instead of `Integer.MAX_VALUE`

They point to the same value, but this change can help a bit the reader to understand the choice behind the choice of the `lengthLimit`.